.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,3 +30,9 @@
     - [ ] N/A or can only be tested interactively
```

Comment:
@TylerLeonhardt, your last commit had 1 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Set-Service can change credentials of a service
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 184
184:             $service | Should -Not -BeNullOrEmpty
```

Reply:
This is external